packages/rich-text/src/plugins/EmbeddedEntityInline/FetchingWrappedInlineEntryCard.tsx

```diff
@@ -109,6 +110,7 @@ interface FetchingWrappedInlineEntryCardProps {
 export function FetchingWrappedInlineEntryCard(props: FetchingWrappedInlineEntryCardProps) {
   const { data: entry, status: requestStatus } = useEntity<Entry>('Entry', props.entryId);
   const { getEntityScheduledActions } = useEntityLoader();
+  const locales = useLocalesContext();
```

Comment:
Not sure why we need this, we should have the correct locale already from `props.sdk.field.locale` or is that a different one?

Reply:
That only gives us the locale code, although currently it's only used to match against the fieldStatus so we could change that to just accept `string | string[]` instead

Reply:
yeah sounds good and then rename it to localeCode(s) for the entityHelper 👍

Reply:
then we also don't need contentful-management as dependency :)